app/controllers/Page.scala

```diff
@@ -23,10 +23,18 @@ object Page extends LilaController {
 
   def contact = page("contact")
 
+  def master = page("master")
```

Comment:
I'm thinking of moving all these pages under /help/<page> to avoid cluttering the root url.